docs/developer-guide/project-structure.md

```diff
@@ -144,12 +144,11 @@ All the [Strongbox] modules are organized under the [Strongbox organization].
 The web integration tests are located in the [strongbox-web-integration-tests] project. 
 They are using the [maven-invoker-plugin] to execute various tests 
 against a Strongbox instance which is started for this purpose. These tests start Maven processes via the [maven-invoker-plugin]
-and are literally mimicking Maven behaviour. The outcome of the tests is validated using Groovy scripts.
+and are literally mimicking Maven behavior. The outcome of the tests is validated using Groovy scripts.
 
 ## Packaging Modules
 
-For a much more lightweight build the modules which carry out the actual packaging into assemblies and distributions, 
-these have been extracted into separate projects under the organization.
+For a much more lightweight build the modules, which carry out the actual packaging into assemblies and distributions, have been extracted into separate projects under the organization.
```

Comment:
@carlspring isn't this now obsolete? I thought we are doing it in the `strongbox/strongbox` project and the `strongbox-distribution` module?

Reply:
I am not sure. I believe this would be a question for carlspring.